pulumi/grapl/__main__.py

```diff
@@ -387,6 +387,34 @@ def main() -> None:
             nomad_agent_security_group_id=nomad_agent_security_group_id,
         )
 
+        pulumi.export(
```

Comment:
Are we exporting everything else in this project as single key/value pairs, or consolidating into objects? That is, export an object for, say, "plugin-registry-db" that has "hostname", "port", "username", etc. keys?

Reply:
Well as much as I'd like to do that, the reason why I'm not is because the local postgres Database object doesn't support these attributes.  We could create a plain old python object, but that feels like the same amount of work as just passing in each attribute